xml/admin_security.xml

```diff
@@ -783,6 +745,105 @@ roleRef:
    </example>
   </sect2>
  </sect1>
+
+ <sect1 xml:id="sec.admin.security.certs">
+  <title>Certificates</title>
+  <para>
+   During the installation of &productname;, a CA (Certificate Authority)
+   certificate is generated; that is then used to authenticate and verify all
+   communications. The process also creates and distributes client certificates
+   for the components.
+  </para>
+  <para>
+   Communication is secured with TLS v1.2 using the <literal>AES 128 CBC</literal>
+   cipher.
+  </para>
+  <para>
+   All client certificates are 4096 Bit RSA encrypted.
+  </para>
+  <para>
+   Certificates are located in
+   <filename>/etc/pki</filename>
+   on each cluster node.
+  </para>
+
+  <sect2 xml:id="sec.admin.security.certs.renewal">
+   <title>Certificate Renewal</title>
+   <para>
+    The CA certificate is valid for <literal>3650</literal> days (10 years)
+    by default.
+   </para>
+   <para>
+    The client certificates are valid for <literal>365</literal> days (1 year)
+    by default.
+   </para>
+   <para>
+    All certificates have a renewal period of <literal>90</literal> days before
+    expiration. If orchestration of the cluster is run during that period, the
+    certificates which are about to expire are renewed automatically.
+   </para>
+   <para>
+    To manually renew certificates, refer to <xref linkend="sec.admin.troubleshooting.replace_certificates"/>.
+   </para>
+   <important>
+    <title>Renewing Expired Certificates</title>
+    <para>
+     If for whatever reason any of the certificates have failed to renew
+     automatically, please log in to &dashboard; and navigate to
+     <guimenu>Settings</guimenu>. Click the <guimenu>Apply changes</guimenu>
+     button. This will force a refresh of the cluster settings and any expired
+     certificates should be renewed automatically.
+    </para>
+    <para>
+     If this still fails, you can replace the certificates manually. Refer to:
+     <xref linkend="sec.admin.troubleshooting.replace_certificates"/>.
+     </para>
```

Comment:
i can see some indentation issues here, but i guess this doesnt make a technical difference ;-)

Reply:
I will fix this before merge, thx